### ISSUE TITLE
Typos on `az cosmosdb --help`

### ISSUE THREAD

**Issue by schradert** [Azure/azure-cli#15524]
**Describe the bug**

Several subgroups have misspelled Cosmos DB as Comsos DB (harmless mistake but noticeable!), namely:
1. keys
2. network-rule
3. private-endpoint-connection
4. private-link-resource

![image](https://user-images.githubusercontent.com/9901294/96036072-d033ed00-0e18-11eb-86ea-2bf27e88d2de.png)

**To Reproduce**
`az cosmosdb --help`

**Expected behavior**
All subgroup `--help` descriptions should say `... Cosmos DB ...`

**Environment summary**
```
az version
{
  "azure-cli": "2.5.1",
  "azure-cli-command-modules-nspkg": "2.0.3",
  "azure-cli-core": "2.5.1",
  "azure-cli-nspkg": "3.0.4",
  "azure-cli-telemetry": "1.0.4",
  "extensions": {}
}
```

**Comment by shurd** [comment]
Thank you for reporting this. I have submitted a fix.

**Comment by yonzhan** [comment]
cosmosdb

**Comment by shurd** [comment]
Fixed in 2.14.0
